packages/web-runtime/src/pages/account.vue

```diff
@@ -314,7 +314,7 @@ export default defineComponent({
       }
 
       try {
-        const { data } = yield* call(clientService.graphAuthenticated.users.getMe())
+        const data = yield* call(clientService.graphAuthenticated.users.getMe())
```

Comment:
Could be one line now:

`graphUser.value = yield* call(clientService.graphAuthenticated.users.getMe())`